src/base/kaldi-error.cc

```diff
@@ -221,7 +221,7 @@ FatalMessageLogger::FatalMessageLogger(LogMessageEnvelope::Severity severity,
   }
 }
 
-[[ noreturn ]] FatalMessageLogger::~FatalMessageLogger() noexcept(false) {
+FatalMessageLogger::~FatalMessageLogger [[ noreturn ]] () noexcept(false) {
```

Comment:
@daanzu would you mind making a PR to make it more like the following, so we can support both compiler versions?  Obviously test that it works first.

```
#if defined(_MSC_VER)
FatalMessageLogger::~FatalMessageLogger [[ noreturn ]] () noexcept(false) 
#else
[[noreturn]] FatalMessageLogger::~FatalMessageLogger () noexcept(false) 
#endif
{
```

Reply:
👍